src/components/AddAsset.tsx

```diff
@@ -112,60 +113,88 @@ export const AddAsset = ({ onClose }: { onClose: () => void }) => {
 
         <Input
           id="aa-asset-code"
-          label="Asset code"
+          // TODO: change type in SDS
+          // @ts-ignore
+          label={
+            <DetailsTooltip
+              details={
+                <>
+                  Assets are identified by 1) their code and 2) either a home
+                  domain or the public key of the issuing account.{" "}
+                  <TextLink href="https://developers.stellar.org/docs/issuing-assets/publishing-asset-info/">
```

Comment:
Out of scope for this PR - but at some point it might be good to create a const called `DEVELOPER_DOCS_URL` or something so we don't need to write out `https://developers.stellar.org/docs/` for every url